toolkit/pallets/block-production-log/src/mock.rs

```diff
@@ -62,13 +62,26 @@ impl frame_system::Config for Test {
 	type PostTransactions = ();
 }
 
+#[cfg(feature = "runtime-benchmarks")]
+pub struct PalletBlockProductionLogBenchmarkHelper;
+
+#[cfg(feature = "runtime-benchmarks")]
+impl crate::benchmarking::BenchmarkHelper<[u8; 32]> for PalletBlockProductionLogBenchmarkHelper {
+	fn producer_id() -> [u8; 32] {
+		Default::default()
+	}
```

Comment:
empty producer_id is considered valid?

Reply:
It is not empty. It is 32 bytes of zero value.

Reply:
yep, but this "zero" filled array which will be used will be considered valid?

Reply:
Yes, this is just to have some similar space occupation.